geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloRasterStore.scala

```diff
@@ -137,6 +135,26 @@ class AccumuloRasterStore(val connector: Connector,
     }
   }
 
+  def getResToBoundsMap: ImmutableMap[Double, BoundingBox] = AccumuloRasterStore.extentCache.get(tableName, resToBoundsCallable)
+
+  def resToBoundsCallable = new Callable[ImmutableMap[Double, BoundingBox]] {
+    override def call(): ImmutableMap[Double, BoundingBox] = {
+      val kvs = metaScanner().toVector
+      val resolutions = kvs.map(_.getKey.getColumnQualifier.toString).map(lexiDecodeStringToDouble)
+      val bounds = kvs.map(_.getValue).map(valueToBbox)
+      val m = new ImmutableMap.Builder[Double, BoundingBox]()
+      (resolutions zip bounds).foreach(x => m.put(x._1, x._2))
```

Comment:
If that makes sense and you like it, feel free to modify the similar block above.